atomic-carousel.twig

```diff
@@ -1,8 +1,14 @@
 {% set _modifiers = carousel.modifiers|default([])|merge([carousel.options.items == 1 ? 'single' : '']) %}
 <div class="owl-carousel {{ _modifiers|join(' ') }}" data-carousel='{{ carousel.options ? carousel.options|json_encode() : "" }}'>
   {% for item in carousel.items %}
-  <div>
-    {{ item }}
-  </div>
+    {% if item.attributes %}
+      <div>
```

Comment:
Yes! this is a way to go. also we can render attributes here, right on `<div>`.

Reply:
yes i thought about that, but then i was thinking it might be cool if output of attributes would be optional ... just guessing.